### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by pacomanval** [Lumieducation/Lumi#1290] [Q&A]
Hi friends, i install Lumi on Mac but appear the next error "editorInstance of h5p editor not defined", can you help me?

**Comment by JPSchellenberg** [comment]
Hey, which version of Lumi are you using?

**Comment by pacomanval** [comment]
Hi, te version Is 0.5.1 on MacOS Mojave

Regards

**Reply by JPSchellenberg** [reply]
Ok, this error seems to come from https://github.com/Lumieducation/H5P-Nodejs-library/blob/5f770a4bdfbac3d8b72bd31ca23340b9285a2d3c/packages/h5p-webcomponents/src/h5p-editor.ts

Is there any Stacktrace you could provide?

Otherwise we will release v0.6.0 soon, which contains a bug-tracking service called sentry, that could provide us with a full stack trace.

**Comment by pacomanval** [comment]
Another error attached

**Comment by sr258** [comment]
Can you tell us detailed steps of what you do before the error occurs?

**Comment by pacomanval** [comment]
Downloaded the version, i run the dmg and when i Select new or open, the
error appear, the only version that i can use Is the 0.3 whitout an error

Regards
